docs/source/json_parser.py

```diff
@@ -0,0 +1,33 @@
+import os
+import json
+
+ROOT_DIR = os.getcwd()
+SOURCE_DIR = os.path.join(ROOT_DIR, "docs/source/")
```

Comment:
```suggestion
from pathlib import Path

SOURCE_DIR = Path(__file__).parent
```

`os.getcwd()` gets the directory you are located in, i.e., that you run the script from. So, this code only works if you run the script inside the root directory.

Generally, using `pathlib` instead of `os.path` is a better approach. It's a more modern library.

Reply:
Thank you